app/uk/gov/hmrc/exports/controllers/SubmissionController.scala

```diff
@@ -128,10 +128,7 @@ class SubmissionController @Inject()(
     submissionRepository.findByEori(eori)
 
   private def notificationHelper(conversationId: String): Future[Int] =
```

Comment:
This method name is completely non-explanatory. Could you change the name to reflect what's being done?

Reply:
Yes, with next PR with logic